src/cascadia/TerminalSettings/TerminalSettings.cpp

```diff
@@ -65,6 +66,16 @@ namespace winrt::Microsoft::Terminal::Settings::implementation
         _colorTable[index] = value;
     }
 
+    hstring TerminalSettings::TabTitle()
+    {
+        return _tabTitle;
+    }
+
+    void TerminalSettings::TabTitle(hstring const& value)

```

Comment:
should be `const hstring&` here and below

Reply:
These things are autogenerated by c++/winrt; they'll be hard to stop from introducing.